datafusion/sql/src/query.rs

```diff
@@ -78,6 +81,76 @@ impl<S: ContextProvider> SqlToRel<'_, S> {
                 let plan = self.order_by(plan, order_by_rex)?;
                 self.limit(plan, query.limit_clause, planner_context)
             }
+        }?;
+
+        self.pipe_operators(plan, pipe_operators, planner_context)
```

Comment:
```suggestion
        self.pipe_operators(plan, query.pipe_operators, planner_context)
```

And can remove the `pipe_operators` variable above.

Reply:
Fixed in [dbcc0c0](https://github.com/apache/datafusion/pull/17278/commits/dbcc0c0d4bdd8effae6a099cbffc0f220dad2846)